cves/CVE-2019-14235.yml

```diff
@@ -206,8 +196,10 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: true
+  note: The method that calls the method that contains the vulnerability is a converter from URI to IRI. IRI means
+    internationalized resource identifier which is a international verison of uri. It contains characters from chinese, korean,
+    and japanese.
```

Comment:
Chinese and Japanese aren't the full set of languages it supports, so maybe add something like "and others".
Also, languages should be capitalized.